test/suites/basedir.bash

```diff
@@ -219,7 +225,14 @@ EOF
     fi
 
     # -------------------------------------------------------------------------
-    for option in MF "MF "; do
+    TEST "-MF/-MQ/-MT with absolute paths"
+
+    if $HOST_OS_WINDOWS; then
+        additional_options=
```

Comment:
Why not test `-MF/path` on Windows?

Also, perhaps don't do this and similar changes since the test suite isn't run on Windows anyway?

Reply:
Where's the logic for skipping tests? What if these tests *can* run on Windows? How do I enable them?

I don't know why the variants without space were excluded. Maybe @rvogg can comment on that.

Reply:
As far as I remember it is because the tests run in the MSYS2 shell. The MSYS runtime automatically converts everything that looks like Unix paths to Windows paths. If there is no space between -MF and the path, it will not be converted. The correct variant under Windows would then probably be -MFC:/msys64/...  but that was too much effort for me to do it cleanly. Also I am not sure anymore if this is accepted by the Windows compiler.

Reply:
> Where's the logic for skipping tests?

Here:

https://github.com/ccache/ccache/blob/c020954d9da3f885eae559a76e6d9f3587f01c5f/test/suites/basedir.bash#L2-L4

Reply:
I prefer to keep these. Looks like they were done to prepare the basedir tests for Windows at a later point.

Reply:
As I mentioned in my original PR, my intention for testing on Windows was to implement the basedir feature for Windows. 
These adaptations come from my side branch where the feature works. 
You can undo the change and re-implement it later if the tests fail, or leave it as is.